open-reac/src/main/java/com/powsybl/openreac/parameters/OpenReacAmplIOFiles.java

```diff
@@ -0,0 +1,87 @@
+import com.powsybl.openreac.parameters.output.OpenReacResult;
+import com.powsybl.openreac.parameters.output.ReactiveSlackOutput;
+
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * OpenReacAmplIOFiles will interface all inputs and outputs needed for OpenReac to the abtracted Ampl Executor.
+ * <p>
+ * The user of OpenReac should not see this class directly. One should use {@link OpenReacParameters} for inputs
+ * and {@link OpenReacResult} for outputs.
+ * However, when adding new inputs (outputs) to OpenReac, one must add {@link AmplOutputFile} (@link AmplInputFile)
+ * here through {@link OpenReacAmplIOFiles#getInputParameters} ({@link OpenReacAmplIOFiles#getOutputParameters})
+ *
+ * @author Nicolas Pierre <nicolas.pierre at artelys.com>
+ */
+public class OpenReacAmplIOFiles implements AmplParameters {
+
+    private final ConstantQGenerators constantQGenerators;
+    private final VariableShuntCompensators variableShuntCompensators;
+    private final VariableTwoWindingsTransformers variableTwoWindingsTransformers;
+    private final AlgorithmInput algorithmParams;
+    private final ReactiveSlackOutput reactiveSlackOutput;
+    private final VoltageLevelLimitsOverrideInput voltageLimitsOverride;
+    private final boolean debug;
+
+    public OpenReacAmplIOFiles(OpenReacParameters params, Network network, boolean debug) {
+        //inputs
+        this.constantQGenerators = new ConstantQGenerators(params.getConstantQGenerators());
+        this.variableShuntCompensators = new VariableShuntCompensators(params.getVariableShuntCompensators());
+        this.variableTwoWindingsTransformers = new VariableTwoWindingsTransformers(params.getVariableTwoWindingsTransformers());
+        this.algorithmParams = new AlgorithmInput(params.getAlgorithmParams());
+        this.voltageLimitsOverride = new VoltageLevelLimitsOverrideInput(params.getSpecificVoltageDelta(), network);
+
+        //outputs
+        this.reactiveSlackOutput = new ReactiveSlackOutput();
+
+        this.debug = debug;
+    }
+
+    public List<ReactiveSlackOutput.ReactiveSlack> getReactiveInvestments() {
+        return reactiveSlackOutput.getSlacks();
+    }
+
+    @Override
+    public Collection<AmplInputFile> getInputParameters() {
+        return List.of(constantQGenerators, variableShuntCompensators, variableTwoWindingsTransformers,
+                algorithmParams, voltageLimitsOverride);
+    }
+
+    @Override
+    public Collection<AmplOutputFile> getOutputParameters(boolean isConvergenceOk) {
+        if (isConvergenceOk) {
+            return List.of(reactiveSlackOutput);
+        }
+        return List.of();
+    }
+
+    /**
+     * Will check that every output file parsing went well.
+     *
+     * @return <code>true</code> if ALL ouput file parsing didn't throw any IOExceptions
+     */
+    public boolean checkErrors() {
+        return !reactiveSlackOutput.isErrorState();
+    }
+
+    // FIXME
+    //@Override
+    public boolean isDebug() {
```

Comment:
To fix after other core PR merge. Here we have a severe issue as AMPL working file are never deleted because is is hard coded on core side that we are in debug mode.